scripts/Crawlers/ZenodoCrawler.py

```diff
@@ -139,6 +140,29 @@ def get_all_dataset_description(self):
             if "keywords" in metadata.keys():
                 keywords = list(map(lambda x: {"value": x}, metadata["keywords"]))
 
+            # Retrieve subject annotations from Zenodo and clean the annotated
+            # subjects to insert in isAbout of DATS file
+            is_about = []
+            species_identifier_source_base_url = "www.ncbi.nlm.nih.gov/taxonomy"
```

Comment:
This URL seems to only be used once in line 150 below. I like the matching idea. One option would be to make this a constant, make it all-caps and put it at the top of the file if we will re-use it later (or to refactor into some place with many of these constants later). If you don't think this will be re-used, I would just put the string in the match statement.

Reply:
Good catch! Indeed, this does not need to be in a variable name. I am not sure whether this will be reused somewhere else so I'll leave it in the code for now. If it is reused by another crawler, then we will create a constant for it. If that makes sense.